From 5b7e57f325321968650676971e40eb7ad9ca6aaa Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Tue, 14 Jul 2015 13:09:51 +0800
Subject: [PATCH] MSVC Builds: Update "Install" Project GUID

To make the Visual Studio project files work together when used in part for
an all-in-one solution file that is used to build from scratch the GTK+
stack, we need to make sure that the GUIDs of the project files in the
solution file do not conflict with each other, as the "install" project
shared the same GUID with Pango's "install" project.
---
 build/win32/vs10/gdk-pixbuf-install.vcxproj |  2 +-
 build/win32/vs10/gdk-pixbuf.sln             | 34 ++++++++++++++---------------
 build/win32/vs9/gdk-pixbuf-install.vcproj   |  2 +-
 build/win32/vs9/gdk-pixbuf.sln              | 34 ++++++++++++++---------------
 4 files changed, 36 insertions(+), 36 deletions(-)

diff --git a/build/win32/vs10/gdk-pixbuf-install.vcxproj b/build/win32/vs10/gdk-pixbuf-install.vcxproj
index 46f9449..5f54977 100644
--- a/build/win32/vs10/gdk-pixbuf-install.vcxproj
+++ b/build/win32/vs10/gdk-pixbuf-install.vcxproj
@@ -19,7 +19,7 @@
     </ProjectConfiguration>
   </ItemGroup>
   <PropertyGroup Label="Globals">
-    <ProjectGuid>{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}</ProjectGuid>
+    <ProjectGuid>{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}</ProjectGuid>
     <RootNamespace>gdkpixbufinstall</RootNamespace>
     <Keyword>Win32Proj</Keyword>
   </PropertyGroup>
diff --git a/build/win32/vs10/gdk-pixbuf.sln b/build/win32/vs10/gdk-pixbuf.sln
index 218fdd0..6e7cd44 100644
--- a/build/win32/vs10/gdk-pixbuf.sln
+++ b/build/win32/vs10/gdk-pixbuf.sln
@@ -5,7 +5,7 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf", "gdk-pixbuf.vc
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf-csource", "gdk-pixbuf-csource.vcxproj", "{3689C772-49A9-4E32-8161-686E1381ADB4}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf-install", "gdk-pixbuf-install.vcxproj", "{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf-install", "gdk-pixbuf-install.vcxproj", "{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}"
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf-query-loaders", "gdk-pixbuf-query-loaders.vcxproj", "{7212330C-14DF-4815-9E3B-4BDB3D21F665}"
 EndProject
@@ -55,22 +55,22 @@ Global
 		{3689C772-49A9-4E32-8161-686E1381ADB4}.Release_GDI+|Win32.Build.0 = Release|Win32
 		{3689C772-49A9-4E32-8161-686E1381ADB4}.Release_GDI+|x64.ActiveCfg = Release|x64
 		{3689C772-49A9-4E32-8161-686E1381ADB4}.Release_GDI+|x64.Build.0 = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|Win32.ActiveCfg = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|Win32.Build.0 = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|x64.ActiveCfg = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|x64.Build.0 = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|Win32.ActiveCfg = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|Win32.Build.0 = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|x64.ActiveCfg = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|x64.Build.0 = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|Win32.ActiveCfg = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|Win32.Build.0 = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|x64.ActiveCfg = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|x64.Build.0 = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|Win32.ActiveCfg = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|Win32.Build.0 = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|x64.ActiveCfg = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|x64.Build.0 = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|Win32.ActiveCfg = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|Win32.Build.0 = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|x64.ActiveCfg = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|x64.Build.0 = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|Win32.ActiveCfg = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|Win32.Build.0 = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|x64.ActiveCfg = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|x64.Build.0 = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|Win32.ActiveCfg = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|Win32.Build.0 = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|x64.ActiveCfg = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|x64.Build.0 = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|Win32.ActiveCfg = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|Win32.Build.0 = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|x64.ActiveCfg = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|x64.Build.0 = Release|x64
 		{7212330C-14DF-4815-9E3B-4BDB3D21F665}.Debug|Win32.ActiveCfg = Debug|Win32
 		{7212330C-14DF-4815-9E3B-4BDB3D21F665}.Debug|Win32.Build.0 = Debug|Win32
 		{7212330C-14DF-4815-9E3B-4BDB3D21F665}.Debug|x64.ActiveCfg = Debug|x64
diff --git a/build/win32/vs9/gdk-pixbuf-install.vcproj b/build/win32/vs9/gdk-pixbuf-install.vcproj
index 45b60f8..3707efc 100644
--- a/build/win32/vs9/gdk-pixbuf-install.vcproj
+++ b/build/win32/vs9/gdk-pixbuf-install.vcproj
@@ -3,7 +3,7 @@
 	ProjectType="Visual C++"
 	Version="9.00"
 	Name="gdk-pixbuf-install"
-	ProjectGUID="{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}"
+	ProjectGUID="{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}"
 	RootNamespace="gdkpixbufinstall"
 	Keyword="Win32Proj"
 	TargetFrameworkVersion="131072"
diff --git a/build/win32/vs9/gdk-pixbuf.sln b/build/win32/vs9/gdk-pixbuf.sln
index d0a0c43..3fb3e06 100644
--- a/build/win32/vs9/gdk-pixbuf.sln
+++ b/build/win32/vs9/gdk-pixbuf.sln
@@ -8,7 +8,7 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf-csource", "gdk-p
 		{01ADE22D-F6EC-4D09-8FE1-B9A97DEC2BA3} = {01ADE22D-F6EC-4D09-8FE1-B9A97DEC2BA3}
 	EndProjectSection
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf-install", "gdk-pixbuf-install.vcproj", "{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "gdk-pixbuf-install", "gdk-pixbuf-install.vcproj", "{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}"
 	ProjectSection(ProjectDependencies) = postProject
 		{7212330C-14DF-4815-9E3B-4BDB3D21F665} = {7212330C-14DF-4815-9E3B-4BDB3D21F665}
 		{01ADE22D-F6EC-4D09-8FE1-B9A97DEC2BA3} = {01ADE22D-F6EC-4D09-8FE1-B9A97DEC2BA3}
@@ -86,22 +86,22 @@ Global
 		{2AD2FF07-0514-41A8-8A32-920C744EA1A6}.Release_GDI+|Win32.Build.0 = Release|Win32
 		{2AD2FF07-0514-41A8-8A32-920C744EA1A6}.Release_GDI+|x64.ActiveCfg = Release|x64
 		{2AD2FF07-0514-41A8-8A32-920C744EA1A6}.Release_GDI+|x64.Build.0 = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|Win32.ActiveCfg = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|Win32.Build.0 = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|x64.ActiveCfg = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug|x64.Build.0 = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|Win32.ActiveCfg = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|Win32.Build.0 = Debug|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|x64.ActiveCfg = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Debug_GDI+|x64.Build.0 = Debug|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|Win32.ActiveCfg = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|Win32.Build.0 = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|x64.ActiveCfg = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release|x64.Build.0 = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|Win32.ActiveCfg = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|Win32.Build.0 = Release|Win32
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|x64.ActiveCfg = Release|x64
-		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7073FB}.Release_GDI+|x64.Build.0 = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|Win32.ActiveCfg = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|Win32.Build.0 = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|x64.ActiveCfg = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug|x64.Build.0 = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|Win32.ActiveCfg = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|Win32.Build.0 = Debug|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|x64.ActiveCfg = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Debug_GDI+|x64.Build.0 = Debug|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|Win32.ActiveCfg = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|Win32.Build.0 = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|x64.ActiveCfg = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release|x64.Build.0 = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|Win32.ActiveCfg = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|Win32.Build.0 = Release|Win32
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|x64.ActiveCfg = Release|x64
+		{FC5AADB5-95CD-4BF0-BA8B-0C16FE7074FB}.Release_GDI+|x64.Build.0 = Release|x64
 		{7212330C-14DF-4815-9E3B-4BDB3D21F665}.Debug|Win32.ActiveCfg = Debug|Win32
 		{7212330C-14DF-4815-9E3B-4BDB3D21F665}.Debug|Win32.Build.0 = Debug|Win32
 		{7212330C-14DF-4815-9E3B-4BDB3D21F665}.Debug|x64.ActiveCfg = Debug|x64
-- 
2.3.5.windows.8

